examples/multiple_tables/main.tf

```diff
@@ -24,5 +24,4 @@ module "bigquery" {
   location                    = "US"
   tables                      = var.tables
   dataset_labels              = var.dataset_labels
-  add_udfs                    = true
```

Comment:
Let's replace this with an invocation of the module (so the example functionality remains unchanged). I want to have an example showing the udfs and bq modules together.